README.rst

```diff
@@ -33,6 +33,20 @@ or with *mamba*::
 
     mamba install radis
 
+
+**Remark**
+Depending on your platform you should write 
```

Comment:
you *may* write. There are so many options. For dev, 'pip install -e .' is useful to also install dependencies